src/components/Editor/index.jsx

```diff
@@ -37,6 +37,7 @@ const Editor = (
     menuClassName,
     attachmentsClassName,
     isMenuIndependent = false,
+    isExpandedMenu = false,
```

Comment:
I think it would be better if we have this behaviour: https://vimeo.com/831415996/e0d9f7c564

```suggestion
    isMenuCollapsible = false,
```